llvm-spirv/test/transcoding/SPV_INTEL_function_pointers/fp-in-recusive-type.ll

```diff
@@ -0,0 +1,84 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc --spirv-ext=+SPV_INTEL_function_pointers -o %t.spv
+; RUN: llvm-spirv %t.spv -to-text -o - | FileCheck %s --check-prefix=CHECK-SPIRV
+; RUN: llvm-spirv %t.spv -r -o %t.r.bc
+; RUN: llvm-dis %t.r.bc -o - | FileCheck %s --check-prefix=CHECK-LLVM
+;
+; This test is intended to check the following source code which caused
```

Comment:
This is universal description of the test :)
Could you add more information of use case that lead to crash?

Reply:
Updated the description